app/src/main/kotlin/com/wire/android/ui/home/settings/privacy/PrivacySettingsViewModel.kt

```diff
@@ -73,4 +96,18 @@ class PrivacySettingsViewModel @Inject constructor(
             }
         }
     }
+
+    fun setScreenshotCensoringConfig(isEnabled: Boolean) {
+        viewModelScope.launch {
+            when (withContext(dispatchers.io()) { persistScreenshotCensoringConfig(isEnabled) }) {
```

Comment:
```suggestion
            when (persistScreenshotCensoringConfig(isEnabled)) {
```

The UseCase should change dispatchers if needed. We don't need/should do that from ViewModel level.